src/clr-angular/utils/a11y/aria-live.service.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { PLATFORM_ID } from '@angular/core';
+import { DOCUMENT, isPlatformBrowser } from '@angular/common';
+import { Inject, Injectable, NgZone, OnDestroy } from '@angular/core';
+import { uniqueIdFactory } from '../id-generator/id-generator.service';
+
+export enum AriaLivePoliteness {
+  off = 'off',
+  polite = 'polite',
+  assertive = 'assertive',
+}
+
+/**
+ * Time in milliseconds before inserting the content into the container
+ */
+export const ARIA_LIVE_TICK: number = 100;
+
+@Injectable()
+export class AriaLiveService implements OnDestroy {
+  private ariaLiveElement: HTMLElement;
+  private document: Document;
+  private previousTimeout: ReturnType<typeof setTimeout>;
+  private id: string = `clr-aria-live-element-${uniqueIdFactory()}`;
+
+  constructor(
+    private _ngZone: NgZone,
+    @Inject(DOCUMENT) _document: any,
+    @Inject(PLATFORM_ID) private platformId: Object
+  ) {
+    this.document = _document;
+  }
+
+  private createLiveAriaContainer(): HTMLElement {
+    const ariaLiveElement = this.document.createElement('div');
+
+    ariaLiveElement.setAttribute('id', this.getId());
+    // Use clarity screen reader class to hide the dom element
+    // and fix the scrollbar shake
+    ariaLiveElement.classList.add('clr-sr-only');
+
+    ariaLiveElement.setAttribute('aria-atomic', 'true');
+    ariaLiveElement.setAttribute('aria-live', AriaLivePoliteness.polite);
+
+    this.document.body.appendChild(ariaLiveElement);
+
+    return ariaLiveElement;
+  }
+
+  public announce(message: string | HTMLElement, politeness: AriaLivePoliteness = AriaLivePoliteness.polite): void {
+    // Create aria-live container when needed.
+    if (this.createLiveContainer() === false) {
+      // when there is not ariaLiveElement we could not apply text to nothing so
+      // that's why we gonna break here.
+      return;
+    }
```

Comment:
We could simply call `createLiveAriaContainer` here without having it to return `boolean` value. 

As mentioned above, `createLiveAriaContainer` should handle inside itself whether it should create live aria container element or just exit if there is an already existing aria-live element.